src/components/DownloadModal/index.tsx

```diff
@@ -0,0 +1,426 @@
+import React, { Component } from 'react';
+// tslint:disable-next-line
+import Modal from 'react-modal';
+import { Colors, Styles, blueGradientBgImg } from '../../constants';
+import { TaskButton, TaskButtonProps } from './TaskButton';
+
+// tslint:disable: no-unsafe-any no-any;
```

Comment:
this should be removed too, we should disable these rules throughout the whole file